eduvpn/server.py

```diff
@@ -196,6 +198,13 @@ def support_contact(self) -> List[str]:
 
     @property
     def oauth_login_url(self):
+        print('=' * 20)
+        from pprint import pprint
```

Comment:
i think this for debugging? we should not print to the console, unless its for the CLI.